src/components/ServiceList.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import FilterList from "../components/FilterList";
+import { GetServices, ClearServicesCache } from "../services/ApiService";
+// import {
+//   Card,
+//   CardContent,
+//   CardFooter
+// } from "@baltimorecounty/dotgov-components";
+
+const ServiceList = () => {
+  const [serviceItems, setServiceItems] = useState();
+
+  ClearServicesCache();
+
+  GetServices()
```

Comment:
Huge concern here, this needs to be wrapped in useEffect, in it's current state, this runs infinitely.

```js
useEffect(() => {
//; do you stuff here
}, []);
```
see https://www.robinwieruch.de/react-hooks-fetch-data

I would recommend making this it own's hook, named `useServices`.

where the end results would look something like

```js
[{ services, isLoading}] = useServices();
```
checkout tranisent tax for an example: https://github.com/baltimorecounty/react-transient-tax/blob/integration/src/components/hooks/useFilingTypes.js

Reply:
@martypowell oh yeah that will do that. That component isnt really ready for testing. Its more or less a place holder. But I will add a hook to make sure that doesnt happen in the future.

Reply:
this is done